### DISCUSSION TITLE
Is there an feature to generate an usdt erc20 address with nodejs(from wasm)

### DISCUSSION THREAD

**Discussion by flyingfishzxf** [trustwallet/wallet-core#2795] [Ideas & Feature request]
Hi, wallet-core team.
I recently used the nodejs version of this great project to manage the wallet. And the getAddressByCoin function worked well to get coin's address, but when I were going to generate an usdt erc20 token address, I could not find any detail about how to acheive my target.
I want to know how can I generate the usdt erc20 token or usdt trc20 token with nodejs/wasm of wallet-core? Or it doesn't support this feature now?
Thank you.

**Comment by optout21** [comment]
What do you mean by 'erc20 token address'? In Ethereum ERC20 tokens are stored on the same address as native ETH.

**Reply by flyingfishzxf** [reply]
Sorry, I'm the new to the crypto currency. I'm not familiar with some concept and theory of the crypto.
So whatever tokens based with Ethereum, their address are the same as the native ETH address, is it right?
Are there any article or documents can help me understand this concept? Can you please give me some advices? 
Thank a lot!
